pkg/reconciler/route/route.go

```diff
@@ -317,16 +319,21 @@ func (c *Reconciler) tls(ctx context.Context, host string, r *v1alpha1.Route, tr
 		}
 	}
 
+	wildcardCert, _ := c.certificateLister.Certificates(r.Namespace).Get(r.Namespace)
```

Comment:
I think I'd rather see us use a fixed resource name per namespace rather than the namespace name.  This is consistent with other namespace-defaulted things I'd seen e.g. Service Accounts.

Reply:
By fixed name do you mean something like using the same name for the cert in each namespace?

Reply:
See my other comment, but I'm leaning more towards using the wildcard DNS name over a fixed name, in case of multiple DNS names per-cluster.

Reply:
I went with namespace.domain.com since `*` isn't allowed in names.